src/components/Button.scss

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import "../color-scheme";
+
+.button {
+  width: 100%;
+
+  padding: 1rem;
+  border-radius: 1000px;
+  border: 0;
+
+  background-color: $foreground;
+  color: $background;
+
+  font-size: 14px;
+  font-weight: 500px;
```

Comment:
this is invalid according to https://www.w3schools.com/cssref/pr_font_weight.asp